gitlab-pipeline/stage/build.yml

```diff
@@ -10,6 +10,7 @@ build:client:qemu:
   variables:
     JOB_BASE_NAME: mender_qemux86_64_uefi_grub
     ONLY_BUILD: "true"
+    BUILD_DOCKER_IMAGES: "true"
```

Comment:
I think we should completely remove `ONLY_BUILD` and [this part](https://github.com/mendersoftware/mender-qa/blob/master/gitlab-pipeline/shared/build_and_test_acceptance.yml#L17-L20).

Variables defined here are self-contained to the pipeline, and not visible to the `release_tool`, so as long as it is consistent we can (and should) clean it up.

Reply:
But if we remove this, then enabling `TEST_QEMUX86_64_UEFI_GRUB` will cause the client build to take much longer. It will essentially be tested twice, once in the build job, and once in the test job. Or am I missing something?

Reply:
A bit late on this (good that you merged). You are right, I got confused (again).